app/views/projects/index.html.erb

```diff
@@ -1,2 +1,10 @@
+<%= content_tag :div,
+  id: "projects_data",
+  data: {
+    props: @props.to_json
+  } do %>
+<% end %>
+
+<%= javascript_pack_tag 'projects' %>
```

Comment:
Ohhh, I get it now - this PR does not replace the entire screen with React, just the top part. You'll come back later to add more stuff to props and take over the rest of the screen, right?

I think there's a slightly easier way to write this DIV though, maybe something like this:

```erb
<div id="projects_data" data-props="<%= @props.to_json %>"></div>
```

Or something like that, my ERB skills have atrophied since discovering HAML. 😝